### PR TITLE
Re-Rebalance Bionic Monster

### PR DESCRIPTION
**Pull request by Grave-Accent**
#### Summary
Buffed Bionic Monster by removing Leaky Bionic and Wire-Induced Stiffness, and removed its Noise-Cancelling Headgear because it was no longer needed.

#### Purpose of change
At some point after the 0.C Cooper build of DDA, some professions were re-balanced, including Bionic Monster, Bionic Monster used to cost no points at all, its upsides counteracted by the many faulty bionics it started with, including the Leaky Bionic.

When it was re-balanced, Bionic Monster became a 2-point Profession, having been given a lot more powerful Bionics to start with, but also even more Faulty Bionics, still including Leaky Bionic. However there is evidence of this re-balance being poorly thought out, the most obvious being that it still has Noise-Canceling Head Gear, which was necessary to sleep because it started with Noisemaker, however the Noisemaker was removed in the re-balance, but the head gear was kept regardless.

Removing Leaky Bionic gets rid of the constant deterioration of the player's health, allowing them to heal from their wounds normally. Whilst removing Wire-Induced Stiffness means that the Bionic Monster will no longer have a permanent 10 Torso Encumbrance, so they are no longer gimped in melee combat, which the Bionic Monster's starting skills and bionics highly encourage.

### CODE REVIEW

**Comment by Zireael07** [comment]
Bionic monster is a challenge start - I'd keep leaky bionic for that reason.

**Comment by Grave-Accent** [comment]
It's not a challenge start for as long as it costs any amount of points, and I prefer this route over making it cheaper again.

**Comment by chaosvolt** [comment]
Yeah, Bionic Monster COSTS 2 points, it doesn't give you 2 points. It shouldn't be crippling.

**Comment by Coolthulhu** [comment]
Leaky is currently rather minor due to health not working properly, making stiffness the only really negative bionic here.
Claws+adrenaline+metabolic is worth those two points, even before skills are taken into account.

I don't see the need for a buff. Bionic Monster is clearly stronger than all the other options at 2 cost. Without stiffness, it would be a clear no-brainer.

**Comment by Grave-Accent** [comment]
Well, then I guess we can consider removing Leaky a buff for the future then, because back when Health still functioned properly, it completely killed one of runs with Bionic Monster, and that was before it was even re-balanced.

As for Stiffness and the skills... yeah, fair, it's the only encumbrance bionic I removed, but it's the only one that affects melee combat.

I could increase the price, lower the skills, or something else. I do want this to be an actual _rebalance_ not only a straight buff. It's just that I've been dissatisfied with Bionic Monster's current iteration since its first rebalance.

File: data/json/professions.json [hunk 1]
```diff
@@ -2703,6 +2703,7 @@
       "bio_voice",
       "bio_ankles",
       "bio_trip",
+	  "bio_stiff",
```
Comment (chaosvolt) [inline data/json/professions.json:2706]: Needs lintin': http://dev.narc.ro/cataclysm/format.html

**Comment by Coolthulhu** [comment]
> I could increase the price, lower the skills, or something else. I do want this to be an actual rebalance and not necessarily just a straight buff.

Ideally, it would have downsides hard enough to matter. But I can't think of any that would be doable without more C++ or non-thematic.
Bumping the cost from 2 to 3 would take it out of multipool "free pick", requiring traits below optimum.
Removing survival and dodge would be a minor thing, but would help.
I'd remove adrenaline bionic. It's thematic, but claws are good enough as they are.

Would be cool if we had more harmful failed bionics, like deafness, craft speed dropped to 20%, permanent bad mood. Then it could be balanced using those.

**Comment by chaosvolt** [comment]
You've mentioned in the discord the idea of mutations gained from faults that can show up from installing a bionic without sterilizing, if we ever implement that we could add some to various bionic classes as an extra source of complications. But that's definitely a future thing since I don't yet know how the implementation would end up working.

**Comment by Grave-Accent** [comment]
Hmm, I'd be willing to learn and perform the C++ required to implement a class trait unique to Bionic Monster.

**Comment by Coolthulhu** [comment]
Will require C++ work for proper rebalancing. For now, closing as stale.

**Pull request closed without merging**